### PR TITLE
Change validator uptime calculation

### PR DESCRIPTION
**Pull request by hanjukim**
## Description
This PR changes calculation of uptime of validator as described in https://github.com/terra-money/station/issues/168

As Is: 1 - missed blocks / 10,000
To be: 1 - missed votes / oracle_slash_window

Fixes: https://github.com/terra-money/station/issues/168

### CODE REVIEW

**Comment by pedro-explore** [comment]
@hanjukim Thank you for this fix and awesome work! 

Do you mind ELI5 why you typed this:

> " // We can ORACLE_SLASH_WINDOW from {lcd}/oracle/parameters, but do this way because it's rare to be changed
>   ORACLE_SLASH_WINDOW: parseInt(ORACLE_SLASH_WINDOW || '100800') || 100800"

but https://lcd.terra.dev/oracle/parameters returns: "slash_window": "432000"?

I still don't understand why 432000 (blockspermonth) is returned but everywhere, including this commit, mentions 100800 (blocksperweek).

**Comment by pedro-explore** [comment]
@hanjukim I hope you understand the confusion we have on discord, if the slashwindow is 1 month why did you make it be 1 week here (100800). 

> It is not wrong. Mainnet parameter has been updated by a proposal
https://github.com/terra-money/fcd/issues/124

**Comment by hanjukim** [comment]
> @hanjukim I hope you understand the confusion we have on discord, if the slashwindow is 1 month why did you make it be 1 week here (100800). 
> 
> > It is not wrong. Mainnet parameter has been updated by a proposal
> https://github.com/terra-money/fcd/issues/124
> 
> 
It is just a default value and it can be changed by editing env var. I dont see any problems here. Thanks

**Comment by fish2plain** [comment]
> > @hanjukim I hope you understand the confusion we have on discord, if the slashwindow is 1 month why did you make it be 1 week here (100800).
> > > It is not wrong. Mainnet parameter has been updated by a proposal
> > > #124
> 
> It is just a default value and it can be changed by editing env var. I dont see any problems here. Thanks

I'm curious why the default MainNet ORACLE_SLASH_WINDOW is not used? 

Per the  getOracleUptime() function you have in your code, which you are using as `uptime`:

If you are using 100800 as ORACLE_SLASH_WINDOW, a validator with a missing 50000 oracle votes would be have uptime of 50.4%; 

But if the default ORACLE_SLASH_WINDOW of 432000 is used, the same validator would have uptime of 88.3%.  

Quite  a big difference.

```
function getOracleUptime(missedOracleVote): number {
  return 1 - Math.min(config.ORACLE_SLASH_WINDOW, missedOracleVote) / config.ORACLE_SLASH_WINDOW
}
```
